ChangeLog

```diff
@@ -22,6 +22,8 @@ Changes
      Inserted as an enhancement for #1371
    * Add support for alternative CSR headers, as used by Microsoft and defined
      in RFC 7468. Found by Michael Ernst. Fixes #767.
+   * Reduced the complexity of the timing tests, as they were failing in less
+     reliable environments.
```

Comment:
I'd rephrase as "as they were assuming more than the underlying OS actually guarantees".